.github/fixtures/config/application.subdir.php

```diff
@@ -0,0 +1,173 @@
+define( 'WP_ENV', env( 'WP_ENV' ) ?: 'production' );
+
+/**
+ * DB settings
+ */
+Config::define( 'DB_NAME', env( 'DB_NAME' ) );
+Config::define( 'DB_USER', env( 'DB_USER' ) );
+Config::define( 'DB_PASSWORD', env( 'DB_PASSWORD' ) );
+Config::define( 'DB_HOST', env( 'DB_HOST' ) ?: 'localhost' );
+Config::define( 'DB_CHARSET', 'utf8mb4' );
+Config::define( 'DB_COLLATE', '' );
+$table_prefix = env( 'DB_PREFIX' ) ?: 'wp_';
+
+if ( env( 'DATABASE_URL' ) ) {
+	$dsn = (object) parse_url( env( 'DATABASE_URL' ) );
+
+	Config::define( 'DB_NAME', substr( $dsn->path, 1 ) );
+	Config::define( 'DB_USER', $dsn->user );
+	Config::define( 'DB_PASSWORD', isset( $dsn->pass ) ? $dsn->pass : null );
+	Config::define( 'DB_HOST', isset( $dsn->port ) ? "{$dsn->host}:{$dsn->port}" : $dsn->host );
+}
+
+/**
+ * Pantheon modifications
+ */
+if ( isset( $_ENV['PANTHEON_ENVIRONMENT'] ) && ! isset( $_ENV['LANDO'] ) ) {
+	Config::define( 'DB_HOST', $_ENV['DB_HOST'] . ':' . $_ENV['DB_PORT'] );
+} else {
+	/**
+	 * URLs
+	 */
+	Config::define( 'WP_HOME', env( 'WP_HOME' ) );
+	Config::define( 'WP_SITEURL', env( 'WP_SITEURL' ) );
+	Config::define( 'DB_HOST', env( 'DB_HOST' ) ?: 'localhost' );
+}
+
+/**
+ * Custom Content Directory
+ */
+Config::define( 'CONTENT_DIR', '/app' );
+Config::define( 'WP_CONTENT_DIR', $webroot_dir . Config::get( 'CONTENT_DIR' ) );
+Config::define( 'WP_CONTENT_URL', Config::get( 'WP_HOME' ) . Config::get( 'CONTENT_DIR' ) );
+
+/**
+ * Authentication Unique Keys and Salts
+ */
+Config::define( 'AUTH_KEY', env( 'AUTH_KEY' ) );
+Config::define( 'SECURE_AUTH_KEY', env( 'SECURE_AUTH_KEY' ) );
+Config::define( 'LOGGED_IN_KEY', env( 'LOGGED_IN_KEY' ) );
+Config::define( 'NONCE_KEY', env( 'NONCE_KEY' ) );
+Config::define( 'AUTH_SALT', env( 'AUTH_SALT' ) );
+Config::define( 'SECURE_AUTH_SALT', env( 'SECURE_AUTH_SALT' ) );
+Config::define( 'LOGGED_IN_SALT', env( 'LOGGED_IN_SALT' ) );
+Config::define( 'NONCE_SALT', env( 'NONCE_SALT' ) );
+
+/**
+ * Multisite
+ */
+Config::define( 'MULTISITE', true );
+Config::define( 'SUBDOMAIN_INSTALL', false );
+Config::define( 'DOMAIN_CURRENT_SITE', defined( 'PANTHEON_HOSTNAME' ) ? PANTHEON_HOSTNAME : $_SERVER['HTTP_HOST'] );
+Config::define( 'PATH_CURRENT_SITE', '/' );
+Config::define( 'SITE_ID_CURRENT_SITE', 1 );
+Config::define( 'BLOG_ID_CURRENT_SITE', 1 );
+Config::define( 'WP_ALLOW_MULTISITE', env( 'WP_ALLOW_MULTISITE' ) ?: true );
+
+/**
+ * Custom Settings
+ */
+Config::define( 'AUTOMATIC_UPDATER_DISABLED', true );
+// Disable the plugin and theme file editor in the admin.
+Config::define( 'DISALLOW_FILE_EDIT', true );
+// Disable plugin and theme updates and installation from the admin.
+Config::define( 'DISALLOW_FILE_MODS', true );
+// Limit the number of post revisions that Wordpress stores (true (default WP): store every revision).
+Config::define( 'WP_POST_REVISIONS', env( 'WP_POST_REVISIONS' ) ?? true );
+
+/**
+ * Debugging Settings
+ */
+if ( $_ENV['PANTHEON_ENVIRONMENT'] === 'dev' || isset( $_ENV['LANDO'] ) ) {
```

Comment:
actually, that code only exists in the fixture. we don't turn debug on in the application.php file (because that's a file that users can do whatever they want with.